config/locales/ja.yml

```
@@ -35,6 +35,7 @@ ja:
     delete: "削除"
     back: "戻る"
     created_at: "作成日"
+    not_specified: "指定無し"
```

細かい話だけど、「なし」でもいいかなと思ってしまった 👀 対応は任せます。

```suggestion
    not_specified: "無し"
```